src/main/java/marquez/api/ErrorResponse.java

```diff
@@ -1,18 +1,14 @@
 package marquez.api;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
 public final class ErrorResponse {
-  private final String error;
-
-  @JsonCreator
-  public ErrorResponse(@JsonProperty("error") final String error) {
-    this.error = error;
-  }
-
   @JsonProperty("error")
-  public String getError() {
-    return error;
-  }
+  private String error;
```

Comment:
If we can get away without using a NoArgsConstructor, then we should make this field `final` (here and every other Lombok object).

Reply:
Please see issue-193.